rental.rb

```diff
@@ -0,0 +1,11 @@
+class Rental
+  def initialize(_date, _book, _person)
```

Comment:
- [ ] I am unable to create a Rental object because of the underscores(`_`) here. I get **`NoMethod`** error. Kindly remove them so that your initialize method looks like so: `def initialize(date, book, person)`